errbot/botplugin.py

```diff
@@ -266,6 +266,19 @@ def callback_message(self, message: Message) -> None:
         """
         pass
 
+    def callback_mention(self, message: Message, mentioned_people: Sequence[Identifier]) -> None:
+        """
+            Triggered if there are mentioned people in message.
```

Comment:
Mentioning how to test if the bot has been mentioned directly here might be interesting.